source/tutorials/metadata.rst

```diff
@@ -0,0 +1,138 @@
+
+Metadata merging is supported anywhere that metadata is accepted within the QIIME 2 ecosystem. For example, it might be interesting to consider coloring an Emperor plot based on the study metadata, or the results of one of the study's alpha diversity metrics. This can be accomplished by providing both the sample metadata mapping file *and* the Faith's PD QZA:
+
+.. TODO: update this link
+.. download::
+   :url: https://docs.qiime2.org/2017.5/data/tutorials/moving-pictures/core-metrics-results/unweighted_unifrac_pcoa_results.qza
+   :saveas: unweighted_unifrac_pcoa_results.qza
+
+.. command-block::
+   qiime emperor plot \
+     --i-pcoa unweighted_unifrac_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-file faith_pd_vector.qza \
+     --o-visualization unweighted-unifrac-emperor-with-alpha.qzv
+
+.. question::
+   What happens to the visualization if the order of the metadata files is reversed in the above ``emperor plot``? What about in the ``metadata tabulate`` example above?
+
+Exploring feature metadata
+--------------------------
+
+As noted above, metadata can be viewed in the context of one of two axes: sample-specific or feature-specific. The previous examples in this tutorial were focused on sample-oriented metadata.
+
+To get started with feature-oriented metadata, first download the example files:
+
+.. TODO: update this link
+.. download::
+   :url: https://docs.qiime2.org/2017.5/data/tutorials/moving-pictures/rep-seqs.qza
+   :saveas: rep-seqs.qza
+
+.. TODO: update this link
+.. download::
+   :url: https://docs.qiime2.org/2017.5/data/tutorials/moving-pictures/taxonomy.qza
+   :saveas: taxonomy.qza
+
+For the sake of illustration, merge the two metadata files, and ``tabulate`` them:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file rep-seqs.qza
+     --m-input-file taxonomy.qza
+     --o-visualization tabulated-feature-assignments.qzv
+
+The resulting table shows the joined metadata files with a column of the the Feature IDs, a column of the representative sequences, and finally a column of the taxonomic assignments .
+
+.. question::
+   Are all QZA files metadata? Are all metadata files QZAs?
+
+Finally, there are additional export options available in the visualizations produced from ``metadata tabulate``. Using the results from ``tabulated-feature-assignments.qzv``, export the data as a new TSV. Open that file in a TSV viewer or text editor and note that the contents are the same as the interactive metadata table in the visualization.
+
+.. question::
+   Can the exported TSV from the above step be used as metadata? What are some benefits of being able to export metadata (hint: see the discussion above about metadata merging)? What about some potential drawbacks (hint: see the overview about provenance_ in QIME 2)?
```

Comment:
*hint: see the overview about provenance_ in QIME 2* -> *hint: what happens to data provenance_ when data is exported from QIIME 2*